src/components/pageLayout/pageLayout.js

```diff
@@ -149,12 +143,6 @@ const PageLayout = ({
   const appLauncherItems = [
     <ApplicationLauncherItem key="application_about" onClick={() => onAbout()}>
       {t('view.label', { context: ['about'] })}
-    </ApplicationLauncherItem>,
-    <ApplicationLauncherItem key="application_install" href={`${(!helpers.DEV_MODE && '.') || ''}/docs/install.html`}>
-      {t('view.label', { context: ['guides-install'] })}
-    </ApplicationLauncherItem>,
-    <ApplicationLauncherItem key="application_using" href={`${(!helpers.DEV_MODE && '.') || ''}/docs/use.html`}>
-      {t('view.label', { context: ['guides-using'] })}
```

Comment:
optional: 

since they're not used anywhere else recommend we also remove the the strings associated with this update.
 - https://github.com/quipucords/quipucords-ui/blob/master/public/locales/en.json#L389-L390

> the CONTRIBUTING.md update unfortunately highlights why #219 is needed =)

Reply:
Right, done in 35aac1d78c01639bdb270deef19cb4b59b72a5a5